go.mod

```diff
@@ -4,26 +4,26 @@ go 1.19
 
 require (
 	github.com/MakeNowJust/heredoc v1.0.0
-	github.com/containers/buildah v1.31.0
-	github.com/containers/common v0.55.1
-	github.com/containers/image/v5 v5.26.1
-	github.com/containers/storage v1.48.0
-	github.com/docker/distribution v2.8.2+incompatible
-	github.com/fsouza/go-dockerclient v1.9.7
-	github.com/go-logr/logr v1.2.4 // indirect
+	github.com/containers/buildah v1.33.8
```

Comment:
/hold

We can't do this in 4.14 without the same code existing in 4.15. Otherwise customers will downgrade buildah whenthey upgrade OCP.